compiler/dyno/tools/chpldoc/chpldoc.cpp

```diff
@@ -71,7 +72,6 @@ std::string outputDir_;
 bool textOnly_ = false;
 std::string CHPL_HOME = getenv("CHPL_HOME");
```

Comment:
Wow, I didn't know you doing write this in C++. But IMO it would be nicer to set this in `main`. Could you update it?